product_docs/docs/biganimal/release/getting_started/preparing_cloud_account/01_preparing_azure/01_understanding_qotas_in_azure.mdx

```diff
@@ -41,30 +37,26 @@ The default public IP addresses limits for basic and standard type are set to 10
 
 ### Recommended limit
 
-EDB recommends that you increase the public IP addresses limits for Basic and Standard type to 50. If you need more than 50 clusters, increase the limit to the number of clusters you plan to deploy.
+This is decided by the number of the clusters you will create. 
```

Comment:
Instead of keeping this section, how about we add the following sentence after the end of the previous paragraph: 

>If you need more than 10 clusters, increase the limit to the number of clusters you plan to deploy.

If we do that I think we can drop the "Default Azure limit" heading.

Reply:
Sure, I have updated it. The new limit needs to consider the current usage. So I changed it to:

> If you need more than 10 clusters, increase the limit to the number of clusters you plan to deploy plus current usage.